clients/privacy-center/public/fides-js-demo.html

```diff
@@ -177,6 +177,30 @@ <h2>Property id</h2>
       } else {
         window.addEventListener("FidesInitialized", onInitialized);
       }
+
+      window.addEventListener("FidesUIShown", () => {
+        // Log event timing
+        const fidesLoaded = performance
+          .getEntriesByType("resource")
+          .find((entry) => entry.name.includes("fides"));
+        const fidesEvents = performance
+          .getEntriesByType("mark")
```

Comment:
same here- can we rely on this existing?

Reply:
worst case it's empty `[]`